admin-manual/maintenance/web-analytics.rst

```diff
@@ -223,31 +197,16 @@ Let's begin!
 Google Analytics web property setup
```

Comment:
Do they still call it web property or should this section title be changed?

Reply:
good catch! thanks